earth_enterprise/src/fusion/autoingest/StorageManager.h

```diff
@@ -286,5 +289,61 @@ bool StorageManager<AssetType>::SaveDirtyToDotNew(
   return true;
 }
 
+template<class Asset>
+Asset Find(const std::string & ref, const AssetDefs::Type & type, const std::string & subtype)
+{
+  notify(NFY_WARN, "Find: %s\t%s\t%s", ref.c_str(), ToString(type).c_str(), subtype.c_str());
```

Comment:
I assume these notify statements are for debugging. We should remove them now (all instances in all 3 functions).